webapp/components/_new/generic/BaseIcon/BaseIcon.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <span v-if="svgIcon" class="base-icon">
+    <component :is="svgIcon" aria-hidden="true" focusable="false" class="svg" />
+  </span>
+</template>
+
+<script>
+import icons, { iconNames } from '~/assets/_new/icons'
+
+export default {
+  props: {
+    name: {
+      type: String,
+      required: true,
+      validator(value) {
+        return iconNames.indexOf(value) !== -1
```

Comment:
```suggestion
        return iconNames.includes(value)
```
could we use this? @roschaefer refactored some legacy code I copied and pasted in to use `includes` instead and it's easier to understand... even for non-developers.

Reply:
Yes, great catch! 💚